scripts/remove

```diff
@@ -13,47 +13,53 @@ source /usr/share/yunohost/helpers
 #=================================================
 # LOAD SETTINGS
 #=================================================
+ynh_print_info "Loading installation settings..."
 
 app=$YNH_APP_INSTANCE_NAME
 
-domain=$(ynh_app_setting_get "$app" domain)
-db_name=$(ynh_app_setting_get "$app" db_name)
+domain=$(ynh_app_setting_get $app domain)
```

Comment:
Please don't remove double quotes. They often are useless for experts, but not using it all the time makes it safer.
You can save time by using https://www.shellcheck.net to do it for you.

Reply:
All double quoted